types/schemaoptions.d.ts

```diff
@@ -49,6 +49,9 @@ declare module 'mongoose' {
     /** Sets a default collation for every query and aggregation. */
     collation?: mongodb.CollationOptions;
 
+    /** Arbitrary options passed to `createCollection()` */
+    collectionOptions?: Record<string, any>;
```

Comment:
Wouldn't it be nice if we could use MongoDB types here instead of `Record<string, any>`?